draft-ietf-quic-qlog-h3-events.md

```diff
@@ -503,10 +505,38 @@ H3DataFrame = {
 
 ### H3HeadersFrame
 
-This represents an *uncompressed*, plaintext HTTP Headers frame (e.g., no QPACK
-compression is applied).
+The payload of an HTTP/3 HEADERS frame is the QPACK-encoding of an HTTP field
+section; see {{Section 7.2.2 of RFC9114}}. `H3HeaderFrame`, in contrast,
+contains the HTTP field section without QPACK encoding.
 
-For example:
+~~~ cddl
+H3HTTPField = {
+    ? name: text
+    ? name_bytes: hexstring
+    ? value: text
+    ? value_bytes: hexstring
+}
+~~~
+{: #h3field-def title="H3HTTPField definition"}
+
+~~~ cddl
+H3HeadersFrame = {
+    frame_type: "headers"
+    headers: [* H3HTTPField]
+}
+~~~
+{: #h3-headersframe-def title="H3HeadersFrame definition"}
+
+For example, the HTTP field section
+
+~~~
+:path: value
```

Comment:
```suggestion
:path: /
```

for consistency with the example below (maybe even better to go for something like /my-resource ?